sampleExtension/optionsPage/htmlFunctionality.js

```diff
@@ -54,7 +68,18 @@ initSubmitWithEnter = function(txtFld) {
 
 initModeSelection = function(buttonGroup) {
     $("input[name=" + buttonGroup + "]").change( function(){
-        var pickedMode = $("input[name=" + buttonGroup + "]:checked").val();
-        setStorageMode(pickedMode);
+        settings_object.setMode($("input[name=" + buttonGroup + "]:checked").val());
+        synchronizer.syncSettings(settings_object);
     });
 };
+
+
+/* -------------------- Interval slider -------------------- */
+
+initIntervalSlider = function() {
+    intervalSlider = new GreenToRedSlider('#interval-slider');
+    intervalSlider.saveValue = function(value) {
```

Comment:
This method is called every "tick" of changing the slider.
A call is made to chrome.sync.storage every call. This makes us exceed the chrome.sync.storage call amount limit. 
So maybe this method should only be called after pressing a "save" button, or only after 2 seconds of no changes.